assets/css/style.css

```diff
@@ -0,0 +1,1003 @@
+  margin-bottom: 6px;
+}
+
+.section-latest {
+  display: flex;
+  flex-direction: row;
+  background-color: rgba(255, 68, 34, 0.06);
+  margin: 2px 0;
+}
+
+.section-latest-header {
+  display: flex;
+  align-items: center;
+  flex-basis: 100%;
+}
+
+.section-latest-title {
+  max-height: 56px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  width: 100%;
+  color: #f42;
+  font-weight: 700;
+  line-height: 1.25;
+  font-size: 16px;
+}
+
+.section-popular-image2 {
+  background-color: #000;
+  background-position: 50%;
+  background-size: cover;
+  height: 96px;
+  width: 96px;
+  background-image: url(https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/08/bitcoin_today-header_bitcoin_today.jpg);
+}
+
+.section-headline {
+  border-bottom: 1px solid #eee;
+}
+
+.section-headline p {
+  display: block;
+  padding: 6px 0 6px 14px;
+  position: relative;
+  color: #555;
+}
+
+/* media queries */
+
+@media (min-width: 768px) {
+  .nav {
+    height: 80px;
+    padding-left: 106px;
+    position: absolute;
+  }
+}
+
+@media (min-width: 1024px) {
+  .nav {
+    padding-left: 0;
+  }
+}
+
+@media (min-width: 1024px) {
+  .nav-logo {
+    -webkit-transform: translateX(-106px);
+    transform: translateX(-106px);
+  }
+}
+
+@media (min-width: 768px) {
+  .nav-logo > a {
+    height: 80px;
+  }
+}
+
+@media (min-width: 768px) {
+  .nav-logo img {
+    -webkit-transform: translateY(16px);
+    transform: translateY(16px);
+  }
+}
+
+@media (min-width: 768px) {
+  .nav-tnw {
+    background: #131313;
+    display: block;
+    font-size: 13px;
+    font-weight: 500;
+    line-height: 32px;
+    max-height: 32px;
+    position: relative;
+    z-index: 2;
+  }
+}
+
+@media (min-width: 768px) {
+  .float-left {
+    float: left;
+    margin-left: 12px;
+    position: relative;
+  }
+}
+
+@media (min-width: 768px) {
+  .nav-tnw a {
+    color: hsla(0, 0%, 100%, 0.6);
+  }
+}
+
+@media (min-width: 768px) {
+  .nav-tnw::after {
+    clear: both;
+    content: "";
+    display: block;
+  }
+}
+
+@media (min-width: 768px) {
+  .nav-sections {
+    display: block;
+    position: absolute;
+  }
+}
+
+@media (min-width: 768px) {
+  .nav-sections > li {
+    float: left;
+    position: relative;
+  }
+}
+
+@media (min-width: 1024px) {
+  .nav-sections-section a {
+    padding-left: 12px;
+    padding-right: 12px;
+  }
+}
+
+@media (min-width: 768px) {
+  .nav-sections-section a {
+    font-family: Shentox, sans-serif;
+    font-size: 14px;
+    font-weight: 500;
+    text-transform: uppercase;
+    color: #aaa;
+    display: block;
+    height: 48px;
+    letter-spacing: 0.04em;
+    line-height: 1;
+    padding: 18px 10px 0;
+  }
+}
+
+@media (min-width: 480px) {
+  .nav-countdown {
+    position: relative;
+    left: auto;
+    float: right;
+    border-right: 1px solid #eee;
+    padding-right: 8px;
+  }
+}
+
+@media (min-width: 480px) {
+  .nav-countdown-wrapper {
+    -webkit-transform: none;
+    transform: none;
+  }
+}
+
+@media (max-width: 1024px) {
```

Comment:
[OPTIONAL]
This rule (`@media (max-width: 1024px)`) is repeated 8x all over your code, it would be nice if you keep your code DRY and wrap all the rules related to that breakpoint in only one `@media (max-width: 1024px)` declaration